src/App.tsx

```diff
@@ -1,73 +1,109 @@
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import debounce from 'lodash.debounce';
+import cn from 'classnames';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [inputText, setInputText] = useState('');
+  const [selectedPerson, setSelectedPerson] = useState('No selected person');
+  const [titleSelectPerson, setTitleSelectPerson] = useState('');
+  const [hasErrorInput, setHasErrorInput] = useState(true);
+  const [dropdownVisible, setDropdownVisible] = useState(false);
+
+  const applyInput = useCallback(debounce(setInputText, 300), []);
```

Comment:
Fix it 
<img width="1193" alt="image" src="https://github.com/mate-academy/react_autocomplete/assets/64750958/69885ecd-38b3-4020-9fb7-3e9815a5cb08">